src/utils/global/ambassadorCard/ambassadorCard.module.scss

```
@@ -70,6 +70,12 @@
     margin: 0 0.5rem 0.25rem;
   }
 
+  .col {
+    display: inline-flex;
+    flex-direction: column;
+    width: fit-content;
+  }
```

```suggestion
  .info {
    display: flex;
    align-items: center;
    gap: .5rem;
    justify-content: space-between;
    
    .tooltip {
      button,
      svg {
        display: block;
      }
    }
  }
```
Your current approach seemed to have some alignment issues, hopefully this cleans it up (see my other comment with the related JSX changes)